src/jvm/test/org/jetbrains/kotlinx/lincheck/test/transformation/WaitNotifyLockTest.kt

```diff
@@ -32,10 +32,31 @@ class WaitNotifyLockTest : VerifierState() {
     }
 }
 
-private class WaitNotifyLock {
+@ModelCheckingCTest(iterations = 30)
+class NestedWaitNotifyLockTest : VerifierState() {
```

Comment:
Here are several tests now. Please rename the filename correspondingly and add a short comment to each test, explaining it verifies.

Reply:
Done